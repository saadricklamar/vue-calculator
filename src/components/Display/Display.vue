<template>
  <section class="display">
    <input type="text" value="0" />
  </section>
</template>

<script>
export default {
  name: "Display"
};
</script>

<style scoped>
.display {
  display: flex;
  justify-content: center;
}
input {
  height: 100px;
  width: 400px;
  background-color: #42b983;
  color: white;
  font-size: 3rem;
  padding-right: 2%;
  text-align: right;
  margin-top: 5%;
  border-radius: 5px;
}
</style>