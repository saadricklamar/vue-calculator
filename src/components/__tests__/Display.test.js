import { mount } from "@vue/test-utils";
import Display from "../Display/Display";

describe("Display", () => {
  const wrapper = mount(Display);
  it("should match the snapshot", () => {
    expect(wrapper.isVueInstance()).toMatchSnapshot();
  });
});
