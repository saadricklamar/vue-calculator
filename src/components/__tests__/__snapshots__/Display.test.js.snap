// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Display should match the snapshot 1`] = `true`;
